numerics_paper/version2/numerical_chj.tex

```diff
@@ -610,16 +628,24 @@ \subsection{Entropy stabilization}\label{sec:entropy_stable}
 \begin{align*}
     G_{i+1/2} &= \frac{1}{2}\left(\entvar_i+\entvar_{i+1}\right)\bfF_{i+1/2}-\frac{1}{2}(\efp_{i}+\efp_{i+1}),
 \end{align*}
-is the discrete entropy flux.
+is a discretization of the entropy flux.
 
 After plugging \eqref{blended-flux-1} into the condition \eqref{es_cond}, we arrive at the value
 \begin{align}\label{lambda_min}
-  \lambda_{i+1/2}^{\min} = \max\left\{0,\frac{\frac{1}{2}(\entvar_{i+1}-\entvar_{i})\cdot\left[\bff(\bfq_{i+1})+\bff(\bfq_{i})-\sum_p\lambda_{i+1/2}^p\W_{i+1/2}^p\right]-(\efp_{i+1}-\efp_i)}{\frac{1}{2}(\entvar_{i+1}-\entvar_{i})\cdot\sum_p\W_{i+1/2}^p}\right\}.
+  \lambda_{i+1/2}^{\min} 
+  = \max\left\{0,\frac{\frac{1}{2}(\entvar_{i+1}-\entvar_{i})\cdot
+    \left[\bff(\bfq_{i+1})+\bff(\bfq_{i})-\sum_p\lambda_{i+1/2}^{p,\EV}\W_{i+1/2}^{p,\Roe}\right]
+    -(\efp_{i+1}-\efp_i)}
+  {\frac{1}{2}(\entvar_{i+1}-\entvar_{i})\cdot\sum_p\W_{i+1/2}^{p,\Roe}}\right\},
 \end{align}
+where 
+\begin{align*}
+  \lambda_{i+1/2}^{p,\EV}=\theta_{i+1/2}\lambda_{i+1/2}^{\max}+(1-\theta_{i+1/2})|\hat{\lambda}_{i+1/2}^p|.
+\end{align*}
 With this choice, \eqref{blended-flux-1} guarantees \eqref{es_cond}, and therefore \eqref{ent_ineq}.
 Note that here we have used the identities
 \begin{align*}
-|\hat{A}_\iph|(Q_{i+1}-Q_i) & = \sum_p |\hat{\lambda}^p_\iph| \W^{p,\Roe}_\iph, & 
+|\hat{A}_\iph|(Q_{i+1}-Q_i) = \sum_p |\hat{\lambda}^p_\iph| \W^{p,\Roe}_\iph, \qquad
```

Comment:
Please explain this change.

Reply:
Do you mean the alignment? I just felt the terms were too far apart so I changed the spacing between them. We can put it back as it was of course.